src/serde/mod.rs

```diff
@@ -18,8 +18,8 @@ mod test;
 pub use de::node_from_bytes;
 pub use de_br::{node_from_bytes_backrefs, node_from_bytes_backrefs_record};
 pub use de_tree::{parse_triples, ParsedTriple};
-pub use ser::node_to_bytes;
-pub use ser_br::node_to_bytes_backrefs;
+pub use ser::{node_to_bytes, node_to_bytes_limit};
+pub use ser_br::{node_to_bytes_backrefs, node_to_bytes_backrefs_limit};
```

Comment:
I think `node_to_bytes_limit` that returns a result is a good idea. As Arvid mentions, we probably do not need `node_to_bytes_backrefs_limit`. It may be possible to DOS the serialization with certain clvm trees, but if it is, they are unknown at this point, and I don't think out-of-memory will be the result (and definitely not an exponentially large serialization like in the uncompressed case).

Reply:
Hmm, on the other hand, creating this API probably doesn't hurt. There's not really a way to limit output memory any other way.

Reply:
OK, I'll strip `node_to_bytes_backrefs_limit` off from `mod.rs`.

Reply:
Done!

Reply:
Seems like we passed the each other.
Do you think `node_to_bytes_backrefs_limit` should be exposed in this PR?

Reply:
@richardkiss I've re-exposed `node_to_bytes_backrefs_limit`.